pipeline/snapcraft/snapcraft.go

```diff
@@ -143,10 +141,17 @@ func create(ctx *context.Context, folder, arch string, binaries []context.Binary
 		return err
 	}
 
-	snap := filepath.Join(
-		ctx.Config.Dist,
-		ctx.Config.ProjectName+"_"+metadata.Version+"_"+arch+".snap",
-	)
+	var target = buildtarget.Target{OS: "linux", Arch: arch}
+	if strings.Contains(arch, "arm") {
+		target.Arch = "arm"
+		target.Arm = strings.Replace(arch, "arm", "", 1)
```

Comment:
could this transformation be hidden inside ForSnap, instead of a prerequisite step?